dll/win32/ws2_32/src/startup.c

```diff
@@ -237,6 +342,30 @@ WSAStartup(IN WORD wVersionRequested,
     /* Check if all worked */
     if (ErrorCode == ERROR_SUCCESS)
     {
+	/* create events for functio WSAThread */
+        gWSAThreadEvent = CreateEvent( 
```

Comment:
And respect the code styling of the module (including cap letter for first character of the comment strings, etc...)

Reply:
@HBelusca I will review all the comments!
@ThFabba The string will be deleted (because of your last comment) and I will change it to call CreateEventW!